deps/3rdparty/cassowary-0.60/c++/ClSimplexSolver.cc

```diff
@@ -440,6 +440,9 @@ ClSimplexSolver::RemoveConstraintInternal(const ClConstraint *const pcn)
     cerr << "delete@ " << pexpr << endl;
 #endif
     delete pexpr;
+
+    // Delete the ClAbstractVarable that backs marker, else we leak
+    delete marker.get_pclv();
```

Comment:
>marker [](start = 11, length = 6)

wasn't quite sure how these are used coupled with autolayout, we can sync offline.